drivers/scheduler/scheduler.go

```diff
@@ -77,7 +78,7 @@ type Driver interface {
 	InspectVolumes(*Context) error
 
 	// DeleteVolumes will delete a storage volume.
```

Comment:
nit: update comment to say "DeleteVolumes will all storage volumes for the given context"